features/step_definitions/comment_steps.rb

```diff
@@ -118,6 +118,11 @@
   assert w.moderated_commenting_enabled?
 end
 
+Then /^comment moderation should not be enabled on "(.*?)"/ do |work|
```

Comment:
Ambiguous regexp literal. Parenthesize the method arguments if it's surely a regexp literal, or add a whitespace to the right of the `/` if it should be a division.